src/components/Dialogs/PeriodDimension/StartEndDate.js

```diff
@@ -10,7 +10,11 @@ export const StartEndDate = ({ value, setValue }) => {
     const [endDate, setEndDate] = useState(endDateStr)
 
     useEffect(() => {
-        setValue(startDate && endDate ? `${startDate}_${endDate}` : '')
+        if (startDate && endDate) {
```

Comment:
What about this solution, that doesn't use `value` in useEffect (which will raise the exhaustive dependencies warning):

```
    useEffect(() => {
        if (startDate && endDate) {
            setValue(`${startDate}_${endDate}`)
        } else if (!startDate || !endDate) {
            setValue(null)
        }
    }, [startDate, endDate])
```

and correspondingly in PeriodDimension:
```
setValue={(value) => {
    if (value === null && selectedIds.length) {
        updatePeriodDimensionItems([])
    } else if (value && value !== selectedIds[0]) {
        updatePeriodDimensionItems([{ id: value }])
    }
}}
```

This way only PeriodDimension maintains the management control of selectedIds.